tests/README.md

```diff
@@ -92,6 +92,17 @@ Some helpful commands:
     $ oc describe pods
     $ oc log -f cockpit-tests-xxxx
 
+The tests need ```/dev/kvm``` to be accessible to non-root users on each node:
+
+    $ sudo chmod 666 /dev/kvm
+
+Some tests need nested virtualization enabled:
+
+    $ sudo -s
+    # echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
+    # echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
+    # ( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
```

Comment:
This is mirroring what the container setup script already does on the host, and what cockpit's test/vm-prep does (except for the chmod). So this is meant for manual setups? Would it be prudent to just refer to vm-prep?

Just a suggestion though, this is (obviously) fine to go in.